app/uk/gov/hmrc/securemessage/repository/MessageRepository.scala

```diff
@@ -37,7 +38,15 @@ class MessageRepository @Inject() (mongo: MongoComponent)(implicit ec: Execution
       "message",
       mongo,
       Letter.letterFormat,
-      Seq.empty[IndexModel],
+      Seq(
```

Comment:
@kjainsc I think this index should already be there as its the same collection that we use for 'message'. Hope you are aware of it. So, I dont think its required here. Please double check.